src/library/Uno.Cupertino/Uno.Cupertino.csproj

```diff
@@ -29,6 +29,12 @@
 	<ItemGroup>
 		<UpToDateCheckInput Include="**\*.xaml" Exclude="bin\**\*.xaml;obj\**\*.xaml" />
 	</ItemGroup>
+	<ItemGroup>
+	  <None Remove="Styles\Controls\TextBlock.xaml" />
+	</ItemGroup>
+	<ItemGroup>
+	  <UpToDateCheckInput Remove="Styles\Controls\TextBlock.xaml" />
+	</ItemGroup>
```

Comment:
I don't believe this is needed, should be taken care of as part of https://github.com/unoplatform/Uno.Themes/blob/4b8ba72287dc5179b80b76b5bcba5af71427eceb/src/library/Uno.Cupertino/Uno.Cupertino.csproj#L30